task6/rosyk_task6.py

```diff
@@ -0,0 +1,113 @@
+import sqlite3 as sql
+from utils import number_validation_input
+
+
+def numbers_sum():
+    result = 0
+    with open('numbers.txt', encoding='utf8') as file, \
+            open('sum_numbers.txt', 'w', encoding='utf8') as output:
+        for line in file:
+            try:
+                result += float(line)
+            except ValueError:
+                print('file contains something except numbers')
+        output.write(str(result))
+    print(result)
+
+
+def odd_even_write():
+    with open('even_or_odd.txt', 'w', encoding='utf8') as file:
+        number = number_validation_input()
+        file.write(f'{number} is even') if number % 2 == 0 else file.write(f'{number} is odd')
+
+
+def python_posibilities_print():
+    with open('learning_python.txt', encoding='utf8') as file:
+        text = file.readlines()
+    for line in text:
```

Comment:
look through other methods:
https://stackoverflow.com/questions/7731213/print-doesnt-print-when-its-in-map-python